### PR TITLE
feat: Improvements to convo and reply box

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

Various Improvements to convo and reply box
- There is a clear draft button now
- New Email participants now show up at front for easier un-selecting
- Email participants now loads correctly from draft
- New convo popup can be full screened now
- You can't select yourself as participant anymore, it was disabled but pressing it still added yourself

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by BlankParticle** [comment]
* **#681** <a href="https://app.graphite.dev/github/pr/un/inbox/681?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BlankParticle and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx [hunk 1]
```diff
@@ -43,12 +47,30 @@ export function NewConvoSheet() {
...
-          onClick={() => setOpen(false)}
-          className="fixed right-1 top-1">
-          <X className="h-4 w-4" />
-        </Button>
+        <div className="absolute right-1 top-1 flex gap-0 p-1">
+          <Button
+            size={'icon'}
+            variant={'ghost'}
+            onClick={() => {
+              router.push(`/${orgShortcode}/convo/new`);
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx:55]: need to make sure to save the draft before maximizing

Comment (McPizza0) [reply apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx:55]: @BlankParticle

**Review by McPizza0 — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx (hunk 1 above)
Comment (BlankParticle) [reply apps/web/src/app/[orgShortcode]/convo/_components/new-convo-sheet.tsx:55]: the draft is saved every 500ms, if you stop typing and click on the fullscreen button, the draft would be saved by then anyways

**Review by McPizza0 — APPROVED** [review]

**Comment by BlankParticle** [comment]
### Merge activity

* **Aug 15, 6:09 AM EDT**: [Graphite](https://app.graphite.dev/github/pr/un/inbox/681) rebased this pull request as part of a merge.

**Pull request merged**